src/datepicker/datepicker-navigation.ts

```diff
@@ -38,6 +38,15 @@ import {NgbDatepickerI18n} from './datepicker-i18n';
         <span class="ngb-dp-navigation-chevron"></span>
       </button>
     </div>
+    <span class="sr-only" [attr.aria-live]="'polite'" [attr.aria-atomic]="true">
+      <span *ngIf="months.length > 1">
+        <span i18n="@@ngb.datepicker.screenreader.from">from</span>
+        {{ this.getScreenReaderFrom() }}
```

Comment:
Same remark here. Name the method according to what it does, ie `getLongDate`, `getShortMotnh`, anything, but not with ScreenReader inside.

Reply:
Same